spec/clinic.v1.yaml

```diff
@@ -4564,6 +4564,36 @@ components:
       properties:
         messageRef:
           $ref: '#/components/schemas/EHRMatchMessageRef'
+        patients:
+          type: object
+          x-stoplight:
+            id: f40eseluu9qk0
+          description: Patient matching options
+          properties:
+            criteria:
+              type: array
+              x-stoplight:
+                id: 1odrg3q8zzdil
+              description: Performs an "OR" match for each item in the array
+              minItems: 1
+              items:
+                x-stoplight:
+                  id: mgi1k2jyojc4k
+                type: string
+                enum:
+                  - MRN
+                  - MRN_DOB
+                  - DOB_FULLNAME
+            onUniqueMatch:
+              type: string
+              x-stoplight:
+                id: s9ujf5ce4vhc1
+              enum:
+                - ENABLE_REPORTS
+                - DISABLE_REPORTS
+              description: Optional action to be performed when a unique match has been found
```

Comment:
This is such a strange concept to me. Are there docs explaining the need/intent behind this?

Why would someone want to make a change to a patient's records without having a known unique identifier for said patient? Isn't there a larger than necessary risk of applying a change to the wrong record?

Perhaps the situation is a clinic worker that has a note that patient John Doe DOB 1969-01-01 should have this order performed... but I guess they might not know if that is unique or not... Hmm... I suppose that makes sense... They don't have to first search to see if it is unique, they can just put the order through, and if it succeeds, it was unique, and if not, they (presumably) get notified...

Perhaps I've answered my own questions then, but if I'm mistaken, please let me know.